ui/components/app/snaps/tx-insight-warnings/tx-insight-warnings.js

```diff
@@ -16,6 +14,8 @@ import {
   ButtonPrimarySize,
   ButtonVariant,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import {
   AlignItems,
   BackgroundColor,
```

Comment:
No story to easily compare UI but can be confident there are no changes from other examples